miner/worker.go

```diff
@@ -38,6 +38,13 @@ type Agent interface {
 	GetHashRate() int64
 }
 
+const MINING_LOG_AT_DEPTH = 5
```

Comment:
Not sure this should be exported. Either way, we also use camel convention for constants. Please use `miningLogAtDepth`